logdna/resource_view_test.go

```diff
@@ -43,13 +43,13 @@ func TestView_expectAppsError(t *testing.T) {
 		},
 	})
 }
-func TestView_expectCategoryError(t *testing.T) {
+func TestView_expectCategoriesError(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			{
-				Config:      testViewConfigCategoryError(),
-				ExpectError: regexp.MustCompile("Inappropriate value for attribute \"category\": list of string required."),
+				Config:      testViewConfigCategoriesError(),
+				ExpectError: regexp.MustCompile("Inappropriate value for attribute \"categories\": list of string required."),
```

Comment:
```suggestion
				ExpectError: regexp.MustCompile("Inappropriate value for attribute \"categories\": list of strings required."),
```

Reply:
This is the error message that comes back from terraform so unfortunately cannot change it. Agree that it would make more sense for this to be strings too.